newIDE/app/src/EventsSheet/EventsTree/Renderers/ForEachEvent.js

```diff
@@ -103,10 +104,12 @@ export default class ForEachEvent extends React.Component<
             tabIndex={0}
           >
             {objectName ? (
-              `Repeat for each instance of ${objectName}:`
+              <Trans>{`Repeat for each instance of ${objectName}:`}</Trans>
```

Comment:
I'm not sure this will work, this will generate a translation string like this: "{0}", because it's seen as a single variable (inside the brackets { }).
Instead:
`<Trans>Repeat for each instance of {objectName}:</Trans>`

which will be properly parsed as "Repeat for each instance of {0}:"